src/extensions/extensions-store.ts

```diff
@@ -20,11 +21,16 @@ export class ExtensionsStore extends BaseStore<LensExtensionsStoreModel> {
 
   protected state = observable.map<LensExtensionId, LensExtensionState>();
 
+  get enabledExtensions(): string[] {
```

Comment:
Why pollute `ExtensionStore` instead of adding this logic to `getEnabledExtensions`  in extension API?